pyupgrade.py

```diff
@@ -1224,6 +1246,7 @@ def _fix_py3_plus(contents_text):
             visitor.six_type_ctx,
             visitor.six_with_metaclass,
             visitor.super_calls,
+            visitor.encode_calls,
```

Comment:
these are alphabetized, as are the attributes on `Py3PlusVisitor` (~roughly -- though grouped with their relevant private state)

Reply:
Sure